common/src/main/java/org/keycloak/common/Profile.java

```diff
@@ -153,6 +153,22 @@ public enum Feature {
         ACCOUNT2("New Account Management Console", Type.DEFAULT),
         ACCOUNT_API("Account Management REST API", Type.DEFAULT),
         ADMIN_FINE_GRAINED_AUTHZ("Fine-Grained Admin Permissions", Type.PREVIEW),
+        /**
+         * Controls the availability of the Admin REST-API.
+         */
+        ADMIN_API("Admin API", Type.DEFAULT),
+
+        /**
+         * Controls the availability of the legacy admin-console.
+         * Note that the admin-console requires the {@link #ADMIN_API} feature.
+         */
+        @Deprecated
+        ADMIN("Legacy Admin Console", Type.DISABLED_BY_DEFAULT),
```

Comment:
I think we can make enabling `ADMIN` and `ADMIN2` a bit easier if we automatically enable `ADMIN_API`. 

We did that before for the now removed `UPLOAD_SCRIPTS` feature so that the `SCRIPTS` feature was automatically enabled.

Reply:
Good point I'll adapt the code

Reply:
@pedroigor the commit [2739d22](https://github.com/keycloak/keycloak/pull/8369/commits/2739d223518fdefab17c24cc2d9a7b6a4a9bef69)  should do the trick